src/field_theory/abel_ruffini.lean

```diff
@@ -0,0 +1,373 @@
+/-
+Copyright (c) 2020 Thomas Brownng and Patrick Lutz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning and Patrick Lutz
```

Comment:
```suggestion
Authors: Thomas Browning, Patrick Lutz
```
We've scrapped authors lists by script before, it's probably best to avoid "and". :-)

Reply:
Sure. There are lots of other files with "and". Should those "and"s be deleted at some point?

Reply:
Nobody's done it, but a PR doing that will be quite welcome.

Reply:
In the mean time, somebody's done it (-;